src/main/java/seedu/address/model/appointment/ReadOnlyAppointmentList.java

```diff
@@ -0,0 +1,17 @@
+package seedu.address.model.appointment;
+
+import java.util.List;
+
+
+/**
+ * Unmodifiable view of an appointment list
+ */
+public interface ReadOnlyAppointmentList {
+
+    /**
+     * Returns an unmodifiable view of the appointment list.
+     * This list will not contain any duplicate appointments.
+     */
+    List<Appointment> getAppointmentList();
```

Comment:
This should be `ObservableList<Appointment>` to follow ReadOnlyAddressBook's convention

Reply:
This is due to AddressBook use UniquePersonList
we are just doing List<Appointment>
if you want 1.3 we can push for similar design

Reply:
If you are aware then it's fine to keep it as it is. But could you explain why you uses List instead of ObservableList? I didn't really understand your explanation :sweat_smile:

Reply:
return ObservableList is due to it call another method from UniquePersonList of which is a class inside AddressBook

Reply:
Yeah I solved it